hunts/views.py

```diff
@@ -21,11 +22,30 @@ def index(request):
     if request.method == "POST":
         form = HuntForm(request.POST)
         if form.is_valid():
+            google_ids = None
+            if (
+                "google_drive_hunt_folder_url" in form.cleaned_data
+                and "google_sheets_template_file_url" in form.cleaned_data
+            ):
+                hunt_folder_id = utils.extract_id_from_drive_folder_url(
+                    form.cleaned_data["google_drive_hunt_folder_url"]
+                )
+                sheets_template_file_id = utils.extract_id_from_sheets_url(
+                    form.cleaned_data["google_sheets_template_file_url"]
+                )
+                google_ids = GoogleInfo(
+                    hunt_folder_id,
+                    autogenerated_sheets_folder_id,
+                    human_files_folder_id=None,
+                    sheets_template_file_id=None,
```

Comment:
I think you're passing the wrong ones here.

I wonder if actually we could get Smallboard to manage this stuff: upon creating a new hunt, we automatically create the folders for it & copy in our template file